### ISSUE TITLE
[feature request] Enable/disable "Multiplane Overlay"

### ISSUE THREAD

**Issue by thetredev** [ChrisTitusTech/winutil#3107]
### Multiplane Overlay: MPO
"Multiplane Overlay" (MPO) in Windows (https://learn.microsoft.com/en-us/windows-hardware/drivers/display/multiplane-overlay-support) at first sounds like a good thing:
> Multiplane overlay (MPO) support is a WDDM feature that allows the graphics hardware to compose multiple layers of content into a single image that it can then display on a screen. It's essentially a hardware-accelerated method of compositing different "planes" of content - where a plane can be a video, the desktop, an application window, etc. - without having to involve the CPU or use up other system resources to do the blending in software.
>
> The MPO feature is available starting in Windows 8.1 (WDDM 1.3). This article describes how to implement this capability in your driver.

### MPO related issues
But, the reality is that people around the internet and people I am directly in contact with are having issues with MPO. Depending on their specific setup it is either better to leave the feature enabled or turning it off via a registry key: https://www.reddit.com/r/Amd/comments/yvyqc7/disabling_multiplane_overlay_mpo_fixed_all/

### WoW + RDR2 crashes, but GPU drivers are not to blame
I first hand experienced an issue with a friend's rig of the following specs:
- Windows 11
- Radeon RX 5700 XT
- Intel i5-13600KF
- Some ASUS motherboard I can't remember the name/model of

To remove any pre-existing driver issues, I did the following:
1. Download and extract DDU
2. Reboot into safe mode
3. Run DDU to remove everything related to AMD
4. Ensure that Windows Update will *never* install any drivers
5. Reboot normally
6. Run Windows Update
7. After successful Windows Update, wait 20 minutes more to ensure no AMD drivers are being installed
8. Reboot normally
9. Open AMD Adrenaline software and click "Reset shader caches"

Even after ensuring a clean GPU driver setup, the specific issue still presented itself: during gameplay of World of Warcraft Classic (and therefore other games using the same engine I presume) the GPU driver would just crash for seemingly no reason, turning off all the displays:
![image](https://github.com/user-attachments/assets/6cb34ff5-b347-42b6-b1c5-10277f1fa6bf)

Sometimes the displays would light up again after a couple of seconds and WoW would just continue like nothing happened, which I found kinda fascinating to be honest. Other times the displays would stay off, leaving me to assume that Windows couldn't recover the GPU driver (state) or something like that.

We also tried Red Dead Redemption 2 to rule out any WoW engine weirdness. There, sometimes the GPU driver would crash after the game apparently successfully exited to the desktop. Sometimes not. Gameplay was fine though.

### Everything hints at MPO
After doing a little bit of research, I tried disabling XMP, because that sounded reasonable. Maybe the GPU was too and the CPU/RAM/XMP combination "expected" something faster. But neither disabling or enabling XMP solved the issue. There was some post about 4G Decoding / Resizable Bar / Smart Memory Access being the issue, but 1) the GPU in question doesn't even support SMA and 2) same thing as with XMP - enabling/disabling didn't help.

After doing some more research, I found the Reddit post I linked above, added the registry key/value pair just to see what's gonna happen, and rebooted. And would you believe... **no GPU crashes ever since!**

Even NVidia themselves have posted the exact same fix online, albeit the issue manifested itself differently: https://nvidia.custhelp.com/app/answers/detail/a_id/5157/~/after-updating-to-nvidia-game-ready-driver-461.09-or-newer%2C-some-desktop-apps. The contents of the reg file(s) are identical to the registry key/value pair of the Reddit post.

## Describe the solution you'd like
Since there is no clear way of saying that MPO is good or bad, I would love a simple checkbox in the Tweas section. Not sure where it should be located but it should say something like **Disable Multipane Overlay** and a tooltip describing what MPO is and stating that
- The user should try this option if they experience GPU driver crashes, also noting that it *might* cause lower performance during gameplay (at least according to my personal understanding of Microsoft's description of the feature)
- The user should reboot their system after the option is ticked
- The user should revert this option (enabling MPO again) if GPU driver crashes persist

Be aware that Windows feature updates *might* override the specific registry setting. I couldn't check that yet because I literally tested all of this like 2 weeks ago for the first time. I'm not sure if winutil already has a way of dealing with feature updates and registry overrides etc. 

## Describe alternatives you've considered
N/A

## Additional context
N/A

## Issue validation
- [x] I checked for duplicate issues.
- [x] I checked for already existing discussions.
- [x] I checked for an already existing pull request addressing the issue.

**Comment by thetredev** [comment]
Ah dang, there already is issue #564. But it's closed? Why is that?

**Comment by MagmaBro123** [comment]
> Ah dang, there already is issue #564. But it's closed? Why is that?

It was closed due to no answer via Git Actions.

**Comment by thetredev** [comment]
Ah gotcha thanks.

**Comment by Squtzy** [comment]
This would be nice, I already have it disabled via doing a NVCleanstall installation of my Nvidia GPU Driver but having a toggle or such in WinUtil would be lovely.

**Comment by thetredev** [comment]
@Squtzy yeah bet! A toggle would be nice. The harder part would be figuring out the group policy that corresponds to the registry key - the policy would be kept across Windows updates whereas registry keys all by themselves, Windows sometimes happily overrides them as part of a feature/cumulative update.

I tried to Google around but I have yet to find a group policy unfortunately. Maybe a custom AMX file would be required, not sure though.

**Comment by thetredev** [comment]
Not stale

**Comment by thetredev** [comment]
Not stale
